entrypoint.sh

```diff
@@ -15,7 +15,7 @@ else
 fi
 
 poetry build
-poetry config pypi-token.pypi $3
+poetry config pypi-token.$4 $3
```

Comment:
Please move this line of code into the if-else block

Reply:
Done!